worlds/subnautica/Locations.py

```diff
@@ -571,9 +571,35 @@ class LocationDict(TypedDict, total=False):
             'need_laser_cutter': False,
             'position': {'x': 83.2, 'y': -276.4, 'z': -345.5}},
 }
+
 if False:  # turn to True to export for Subnautica mod
-    payload = {location_id: location_data["position"] for location_id, location_data in location_table.items()}
     import json
+    import math
 
+    payload = {location_id: location_data["position"] for location_id, location_data in location_table.items()}
     with open("locations.json", "w") as f:
         json.dump(payload, f)
+
+    def radiated(pos: Vector):
+        aurora_dist = math.sqrt((pos["x"] - 1038.0) ** 2 + (pos["y"] - -3.4) ** 2 + (pos["y"] - -163.1) ** 2)
+        return aurora_dist < 950
+
+    def far_away(pos: Vector):
+        return (pos["x"] ** 2 + pos["z"] ** 2) > (800 ** 2)
```

Comment:
pos["y"] does not matter?

Reply:
depth is checked seperately, so yes it doesn't matter here.